src/java.compiler/share/classes/javax/lang/model/util/Elements.java

```diff
@@ -51,11 +51,18 @@
 public interface Elements {
 
     /**
-     * Returns a package given its fully qualified name if the package is unique in the environment.
-     * If running with modules, all modules in the modules graph are searched for matching packages.
+     * Returns a package given its fully qualified name if the package is uniquely
+     * determinable in the environment.
+     *
+     * If running with modules, packages of the given name are searched in a
+     * two-stage process:
+     * -find non-empty packages with the given name returned by {@link #getPackageElement(ModuleElement, CharSequence)}, where the provided ModuleSymbol is any root module,
```

Comment:
it would be nice to try to keep the line width as close as possible to 80 chars, same for the CSR